### PR TITLE
ENH/TST: Add now backend test

### LINKED ISSUE #1577: TST: Create ALL test for ibis.now()

### PR DESCRIPTION
**Pull request by cpcloud**
Closes #1577

### CODE REVIEW

File: ibis/pandas/dispatch.py [hunk 1]
```diff
@@ -42,8 +42,9 @@ def execute_node_without_scope(node, **kwargs):
 
 
 # Default returns an empty scope
+@pre_execute.register(ops.Node)
 @pre_execute.register(ops.Node, ibis.client.Client)
-def pre_execute_default(node, client, **kwargs):
+def pre_execute_default(node, *clients, **kwargs):
```
Comment (kszucs) [inline ibis/pandas/dispatch.py]: Is this the new variadic dispatch? Or just the first `client` needs to be an instance of `ibis.client.Client`?

Comment (cpcloud) [reply ibis/pandas/dispatch.py]: No, not variadic. `ibis.pandas.execute(ibis.now())` has no client, and yet it still needs to be able to `pre_execute`.

File: ibis/tests/all/test_temporal.py [hunk 2]
```diff
@@ -295,3 +298,33 @@ def test_day_of_week_column_group_by(
...
+@tu.skipif_unsupported
+def test_now(backend, con):
+    expr = ibis.now()
+    result = con.execute(expr)
+    pandas_now = pd.Timestamp('now')
+    assert isinstance(result, pd.Timestamp)
+
+    # this could fail if we're testing in different timezones and we're testing
+    # on Dec 31st
+    assert result.year == pandas_now.year
```
[the remark this answers is not in the document]
Comment (cpcloud) [reply ibis/tests/all/test_temporal.py]: Do you mean as opposed to checking that they are exactly equal?

File: ibis/pandas/dispatch.py (hunk 1 above)
Comment (kszucs) [reply ibis/pandas/dispatch.py]: Ahh, I see now.

File: ibis/tests/all/test_temporal.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (kszucs) [reply ibis/tests/all/test_temporal.py]: As opposed to checking only the year part equality.

**Comment by cpcloud** [comment]
@kszucs I'll make an issue to follow up with a more precise test. The various implementations of `now` are returning timestamps with different timezones, so that will have to be worked out to get a more precise comparison

**Pull request closed without merging**